### PR TITLE
Feature/jjdd8 ism 7

### PR DESCRIPTION
**Pull request by ChrisMcCat**
fighting with pom, maven builds working jar

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: pom.xml [hunk 1]
```diff
@@ -30,6 +31,68 @@
...
+            <goal>copy</goal>
+          </goals>
+          <configuration>
+            <artifactItems>
+              <artifactItem>
+                <groupId>com.infoshareacademy</groupId>
+                <artifactId>is-masters</artifactId>
+                <version>1.0-SNAPSHOT</version>
+                <!--
+                <type>[ packaging ]</type>
```
Comment (rafalmisiak) [inline pom.xml]: nie puszczajmy komentarzy do repo

Comment (rafalmisiak) [reply pom.xml]: komentarzy kodu

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**